airbyte-webapp/src/components/TextWithHTML/TextWithHTML.tsx

```diff
@@ -1,28 +1,31 @@
 import React from "react";
-import { FormattedHTMLMessage } from "react-intl";
 import sanitizeHtml from "sanitize-html";
 
 type IProps = {
   text?: string;
 };
 
+const allowedAttributes = {
+  ...sanitizeHtml.defaults.allowedAttributes,
+  a: [...sanitizeHtml.defaults.allowedAttributes["a"], "rel"],
+};
+
 const TextWithHTML: React.FC<IProps> = ({ text }) => {
   if (!text) {
     return null;
   }
 
   const sanitizedHtmlText = sanitizeHtml(text, {
+    allowedAttributes,
     transformTags: {
-      a: sanitizeHtml.simpleTransform("a", { target: "_blank" }),
+      a: sanitizeHtml.simpleTransform("a", {
+        target: "_blank",
+        rel: "noopener noreferrer",
+      }),
     },
   });
 
-  return (
-    <FormattedHTMLMessage
-      id="textWithHtmlTags"
-      defaultMessage={sanitizedHtmlText}
-    />
-  );
+  return <span dangerouslySetInnerHTML={{ __html: sanitizedHtmlText }} />;
```

Comment:
:information_source: The `FormattedHTMLMessage` component doesn't exist anymore in newer versions of `react-intl`. Instead the `FormattedMessage` supports now all the same HTML. That said, in this place the text/HTML we print is comping directly from the specification, and there is no purpose putting it through `react-intl`, since it would still be a single id (`textWithHtmlTags`), such we can't translate it that way. If we'd want those property descriptions to be internationalized, we'd anyway need to build a system for that. Thus changing this to a regular `dangerouslySetInnerHTML` for now, which is safe in this place, because we used `sanitize-html` beforehand.